transmart-server/grails-app/controllers/org/transmart/server/UrlMappings.groovy

```diff
@@ -0,0 +1,18 @@
+package org.transmart.server
+
+class UrlMappings {
+
+    static mappings = {
+        "/$controller/$action?/$id?(.$format)?"{
+            constraints {
+                // apply constraints here
+            }
+        }
+
+        "/"(controller: 'auth', action: 'index')
```

Comment:
Doesn't this require an AuthController to exist? Navigating to `/` gives an error.